about_styles.css

```diff
@@ -0,0 +1,421 @@
+.mobile-menu .large-text {
+  font-size: 100%;
+  letter-spacing: -2px;
+}
+
+/* end logo */
+
+.mobile-menu li:nth-child(2) a,
+.mobile-menu li:nth-child(7) a {
+  color: #ec5242;
+}
+
+.campaign {
+  border: 4px solid #ec5242;
+  width: fit-content;
+}
+
+.mobile-menu li {
+  padding: 10px;
+  display: flex;
+}
+
+.mobile-menu li:hover {
+  background-color: #eee;
+}
+
+.mobile-menu li a:hover {
+  color: red;
+  font-size: 0.9rem;
+}
+
+/* end common header styles */
+
+.headline {
+  background-color: #f3f3f3;
+  padding-bottom: 64px;
+  font-size: 0.7rem;
+  text-align: center;
+  line-height: 1.5;
+}
+
+h1 {
+  color: #ec5242;
+  font-size: 2rem;
+  margin: 0 auto 30px;
+  font-weight: 900;
+  line-height: 1;
+}
+
+.not-about {
+  display: none;
+}
+
+span.hello {
+  font-size: 1.2rem;
+  font-weight: 300;
+  line-height: 3;
+}
+
+p.intro {
+  border: 2px solid #eee;
+  background: white;
+  margin: 20px;
+  padding: 15px;
+  font-weight: 500;
+}
+
+p.email {
+  text-decoration: underline;
+}
+
+/* Logo Section */
+section.logo {
+  text-align: center;
+  font-size: 0.7rem;
+  line-height: 1.5;
+  border-bottom: 1px solid #eee;
+  padding-top: 30px;
+}
+
+h2 {
+  font-size: 1rem;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+hr {
+  width: 30px;
+  border: 0.1px solid #ec5242;
+  margin: 0 auto 10px;
+}
+
+.big-container {
+  width: 90%;
+  border: 1px solid #eee;
+  margin: 30px auto;
+  padding: 30px;
+}
+
+.make-logo-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: end;
+  width: fit-content;
+  margin: 0 auto;
+}
+
+.make-gear {
+  width: fit-content;
+  color: #ec5242;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.make-gear .big-gear {
+  font-size: 300%;
+}
+
+.make-large-text {
+  font-size: 200%;
+  letter-spacing: -2px;
+}
+
+/* past events */
+section.past-events {
+  text-align: center;
+  font-size: 0.7rem;
+  line-height: 1.5;
+  padding: 30px 0;
+}
+
+.previous-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  margin-top: 30px;
+}
+
+.prev-conf1,
+.prev-conf2 {
+  position: relative;
+  width: 295px;
+  height: 165px;
+  overflow: hidden;
+  background: url("images/previousmeet1.webp") 0 0 no-repeat;
+  background-size: 100%;
+  box-shadow: inset 0 0 0 1000vmax rgba(236, 82, 66, 0.7);
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.prev-conf2 {
+  background: url("images/previousmeet2.jpeg") 0 0 no-repeat;
+  background-size: 100%;
+}
+
+h3 {
+  font-size: 1rem;
+  font-weight: 900;
+}
+
+/* partner section */
+
+.partner {
+  background-color: #272a31;
+  color: #eee;
+  padding: 30px 0;
+}
+
+.companies {
+  font-size: 2rem;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  column-gap: 70px;
+  row-gap: 10px;
+  padding: 15px 30px;
+  color: grey;
+}
+
+footer p {
+  padding: 30px;
+  font-size: 0.7rem;
+}
+
```

Comment:
- Hey, after you implement the changes I suggested concerning adding the logo to your footer, then you should also fix the alignment issue in this mobile version. The logo should be at the left and the text contents should be at the right as seen in the image below to make your design close to the original design. see the images below:


*design*
![Screenshot from 2023-03-16 19-13-02](https://user-images.githubusercontent.com/59397159/225715701-2be43e5c-4d01-4936-bafd-dcb2ad5a0a53.png)

*Your design*
![Screenshot from 2023-03-16 19-17-41](https://user-images.githubusercontent.com/59397159/225715922-655c3ed5-c33b-4189-8817-aa5bbfabb24b.png)